src/modules/blockchainApplication/explore/components/BlockchainApplicationStatistics/index.js

```diff
@@ -79,7 +79,7 @@ const BlockchainApplicationStatistics = () => {
               </Tooltip>
             </div>
             <p className={`${styles.statsInfo} stats-info-value`}>
-              <TokenAmount token="LSK" val={amount} />
+              <TokenAmount Wrapper={React.Fragment} token="LSK" val={amount} />
```

Comment:
```suggestion
              <TokenAmount wrapper={React.Fragment} token="LSK" val={amount} />
```
lets ensure the casing is followed, if you had specific intention to change maybe please propose 🙏🏼

Reply:
It is uppercase to indicate that the prop should be a component/dom node. React components need to be uppercase if used in this way. I havent found a pattern for this in our code, but if we want it lowercase we would have to do this in the component
<img width="333" alt="image" src="https://user-images.githubusercontent.com/8784876/206671425-de34eec1-a1ae-431e-8870-5edd40c65afc.png">

Which is a bit more messy imo. Or what do you think?

Reply:
make sense, I wasn't aware of this, lets keep it as it is.

Reply:
Great, it is nice to have this specific prop in uppercase to indicate this